xarray/plot/plot.py

```diff
@@ -437,7 +448,8 @@ def plotmethod(_PlotMethods_obj, x=None, y=None, ax=None, row=None,
                    col=None, col_wrap=None, xincrease=True, yincrease=True,
                    add_colorbar=True, add_labels=True, vmin=None, vmax=None,
                    cmap=None, colors=None, center=None, robust=False,
-                   extend=None, levels=None, subplot_kws=None, **kwargs):
+                   extend=None, levels=None, subplot_kws=None,
+                   cbar_ax=None, cbar_kwargs={}, **kwargs):
```

Comment:
I'd prefer `cbar_kwargs=None`, matches the `subplot_kws` style and it is generally good not to put mutable objects as kwarg defaults.